lib/Php/ClassGenerator.php

```diff
@@ -37,23 +98,52 @@ private function handleBody(Generator\ClassGenerator $class, PHPClass $type)
         return true;
     }
 
+    /**
+     * @param PHPClass $class
+     * @return bool
+     */
     private function isNativeType(PHPClass $class)
     {
-        return ! $class->getNamespace() && in_array($class->getName(), [
-            'string',
-            'int',
-            'float',
-            'integer',
-            'boolean',
-            'array',
-            'mixed',
-            'callable'
-        ]);
+        return !$class->getNamespace() && in_array(
+            $class->getName(),
+            [
+                'string',
+                'int',
+                'float',
+                'integer',
+                'boolean',
+                'array',
+                'mixed',
+                'callable'
+            ]
+        );
+    }
+
+    /**
+     * @param string $shortDescription
+     * @param string $longDescription
+     * @param array $tags
+     * @return DocBlockGenerator
+     */
+    private function createDocBlockGenerator($shortDescription = null, $longDescription = null, array $tags = [])
+    {
+        $docBlockGenerator = $this->docBlockGeneratorPrototype ?
```

Comment:
Its about DI here. Its not a good practice to instantiate objects on your own in your classes. At least allow to have a setter. This will allow to Unit-Test your application without unknown/unchangable dependencies.